client/src/main/scala/org/http4s/client/PoolManager.scala

```diff
@@ -36,6 +41,30 @@ private final class PoolManager[F[_], A <: Connection[F]](
   private val idleQueues = mutable.Map.empty[RequestKey, mutable.Queue[A]]
   private var waitQueue = mutable.Queue.empty[Waiting]
 
+  private def publishStats = statsPublisher match {
+    case None => F.unit
+    case Some(op) =>
+      F.start(
```

Comment:
lol, not scalafmt's finest moment, is it?  If you import `cats.effect.implicits._`, you'll get syntax that makes this a little less deep.

Reply:
Haha, yeah I thought I could do with neatening this up a bit. It is a bit hadkouken.

![img](https://i.redditmedia.com/I1CShqp2wBguKcuv21m6XRnREKuw8kyAoCavCAGJPBI.jpg?w=720&s=629fe555a576a7a6dd85d04cebbd8c15)